addons/im_livechat/static/tests/call.test.js

```diff
@@ -0,0 +1,40 @@
+import {
+    click,
+    contains,
+    mockGetMedia,
+    setupChatHub,
+    start,
+    startServer,
+} from "@mail/../tests/mail_test_helpers";
+
+import { defineLivechatModels } from "@im_livechat/../tests/livechat_test_helpers";
+
+import { test } from "@odoo/hoot";
+import { mockDate } from "@odoo/hoot-mock";
+
+import { Command, serverState } from "@web/../tests/web_test_helpers";
+
+defineLivechatModels();
+
+test("should display started a call message", async () => {
+    mockDate("2025-01-01 12:00:00", +1);
```

Comment:
`+1` is weird, but somehow the test doesn't pass last assertion although UI is the same... I have no explanation on this different assertion results.